ui/appui-abstract/src/appui-abstract/toolbars/ToolbarItem.ts

```diff
@@ -113,7 +113,7 @@ export interface CustomButtonDefinition extends ToolbarItem {
 export type CommonToolbarItem = ActionButton | GroupButton | CustomButtonDefinition;
 
 /** Type for Toolbar Item Id
- * @deprecated in 3.6.
+ * @deprecated in 3.6. Please use ToolbarItemId from @itwin/appui-react.
```

Comment:
I was planning to get rid of this altogether.
```suggestion
 * @deprecated in 3.6. Please use `ToolbarItem["id"]` from @itwin/appui-react.
```